integrations/linear/linear/queries.py

```diff
@@ -0,0 +1,372 @@
+                        email
+                    }
+                    parent {
+                        id
+                        identifier
+                    }
+                }
+            }
+            pageInfo {
+                hasNextPage
+                startCursor
+                endCursor
+            }
+        }
+    }
+    """,
+    "GET_NEXT_ISSUES_PAGE": """
+    query Issues {
+        issues(first: {{ page_size }}, after: "{{ end_cursor }}") {
+            edges {
+                cursor
+                node {
+                    id
+                    createdAt
+                    updatedAt
+                    archivedAt
+                    number
+                    title
+                    priority
+                    estimate
+                    sortOrder
+                    startedAt
+                    completedAt
+                    startedTriageAt
+                    triagedAt
+                    canceledAt
+                    autoClosedAt
+                    autoArchivedAt
+                    dueDate
+                    slaStartedAt
+                    slaBreachesAt
+                    trashed
+                    snoozedUntilAt
+                    labelIds
+                    previousIdentifiers
+                    subIssueSortOrder
+                    priorityLabel
+                    integrationSourceType
+                    identifier
+                    url
+                    branchName
+                    customerTicketCount
+                    description
+                    descriptionState
+                    team {
+                        id
+                        name
+                        key
+                    }
+                    state {
+                        name
+                    }
+                    creator {
+                        name
+                        email
+                    }
+                    assignee {
+                        name
+                        email
+                    }
+                    parent {
+                        id
+                        identifier
+                    }
+                }
+            }
+            pageInfo {
+                hasNextPage
+                startCursor
+                endCursor
+            }
+        }
+    }
+    """,
+    "GET_FIRST_TEAMS_PAGE": """
+    query Teams {
+        teams(first: {{ page_size }}) {
+            edges {
+                cursor
+                node {
+                    id
+                    name
+                    key
+                    description
+                    organization {
+                        id
+                        name
+                        urlKey
+                    }
+                }
+            }
+            pageInfo {
+                hasNextPage
+                startCursor
+                endCursor
+            }
+        }
+    }
+    """,
+    "GET_NEXT_TEAMS_PAGE": """
```

Comment:
Ill probably want you to make a function that will add the pagination instead of duplicating queries 

Maybe you can pass page:x, after:0?

Reply:
Replaced with single query and templating